src/tooltip.js

```diff
@@ -1,440 +1,327 @@
-            gravity: {get: function(){return gravity;}, set: function(_){gravity=_;}},
-            distance: {get: function(){return distance;}, set: function(_){distance=_;}},
-            snapDistance: {get: function(){return snapDistance;}, set: function(_){snapDistance=_;}},
-            classes: {get: function(){return classes;}, set: function(_){classes=_;}},
-            chartContainer: {get: function(){return chartContainer;}, set: function(_){chartContainer=_;}},
-            fixedTop: {get: function(){return fixedTop;}, set: function(_){fixedTop=_;}},
-            enabled: {get: function(){return enabled;}, set: function(_){enabled=_;}},
-            hideDelay: {get: function(){return hideDelay;}, set: function(_){hideDelay=_;}},
-            contentGenerator: {get: function(){return contentGenerator;}, set: function(_){contentGenerator=_;}},
-            valueFormatter: {get: function(){return valueFormatter;}, set: function(_){valueFormatter=_;}},
-            headerFormatter: {get: function(){return headerFormatter;}, set: function(_){headerFormatter=_;}},
-            keyFormatter: {get: function(){return keyFormatter;}, set: function(_){keyFormatter=_;}},
-            headerEnabled:   {get: function(){return headerEnabled;}, set: function(_){headerEnabled=_;}},
-
-            // internal use only, set by interactive layer to adjust position.
-            _isInteractiveLayer: {get: function(){return isInteractiveLayer;}, set: function(_){isInteractiveLayer=!!_;}},
-
-            // options with extra logic
-            position: {get: function(){return position;}, set: function(_){
-                position.left = _.left !== undefined ? _.left : position.left;
-                position.top  = _.top  !== undefined ? _.top  : position.top;
-            }},
-            offset: {get: function(){return offset;}, set: function(_){
-                offset.left = _.left !== undefined ? _.left : offset.left;
-                offset.top  = _.top  !== undefined ? _.top  : offset.top;
-            }},
-            hidden: {get: function(){return hidden;}, set: function(_){
-                if (hidden != _) {
-                    hidden = !!_;
-                    nvtooltip();
-                }
-            }},
-            data: {get: function(){return data;}, set: function(_){
-                // if showing a single data point, adjust data format with that
-                if (_.point) {
-                    _.value = _.point.x;
-                    _.series = _.series || {};
-                    _.series.value = _.point.y;
-                    _.series.color = _.point.color || _.series.color;
-                }
-                data = _;
-            }},
-
-            // read only properties
-            tooltipElem: {get: function(){return tooltipElem;}, set: function(_){}},
-            id: {get: function(){return id;}, set: function(_){}}
+            positionTooltip();
```

Comment:
Not the tooltip content, but the tooltip position. This is the same as the old version, I only cleaned it up:
https://github.com/novus/nvd3/blob/master/src/tooltip.js#L345